templates/archives/stories/women_in_symbols.jinja2

```diff
@@ -0,0 +1,28 @@
+{% extends 'archives/story.jinja2' %}
+
+{% block story_title %}
+Women in Symbols
+{% endblock %}
+
+{% block story_body %}
+<p>It is often quite difficult to find the work of women in our society’s historical record, particularly in regards to computing. Most of the historical materials we reference to learn about early computing have been signed by men, with little evidence of the women behind them. For example, a majority of the letters in this archive have been written by a secretary, whose initials only appear at the end of the letter, such as in the picture to the right, where FMV has signed the letter and NB has written it up.</p>
```

Comment:
Why do you think it is difficult to surface women's contributions? This is a big question and concerns the entire field of history (etc.), so it may be a good idea to try to scope things down to the current archive and project.